requirements.txt

```diff
@@ -0,0 +1,32 @@
+# Developers: Please do not modify this auto-generated file. If
```

Comment:
Is there any way we can provide a basic check (maybe as a precommit hook) to verify that the auto generated and maintained values correctly match each other?

Reply:
Ah I see there is a check already. Nice!